ios/Plugin/Plugin.swift

```diff
@@ -25,20 +26,30 @@ public class SecureStoragePlugin: CAPPlugin {
     
     @objc func get(_ call: CAPPluginCall) {
         let key = call.getString("key") ?? ""
-        let hasValue = KeychainWrapper.standard.hasValue(forKey: key)
-        if(hasValue) {
+        let hasValueDedicated = keychainwrapper.hasValue(forKey: key)
+        let hasValueStandard = KeychainWrapper.standard.hasValue(forKey: key)
+        if(hasValueDedicated || hasValueStandard) {
             call.success([
-                "value": KeychainWrapper.standard.string(forKey: key) ?? ""
+                "value": hasValueDedicated
+                    ? keychainwrapper.string(forKey: key) ?? ""
+                    : KeychainWrapper.standard.string(forKey: key) ?? ""
             ])
         }
         else {
             call.error("Item with given key does not exist")
         }
     }
     
+    @objc func keys(_ call: CAPPluginCall) {
+        let keys = keychainwrapper.allKeys();
```

Comment:
New keys() method return keys from keychan wrapper, so entires which are still accessible from standard keychain by get method as a fallback, are not listed. I am just thinking if this is not confusing... maybe we can just write a note to README that only entries inserted after specifiv version are listed there on ios plaform. OR maybe you can add entried from standard keychain which match PREFIX.

Reply:
prefix is only available in the web fallback, right? Sadly we can only distinguish between dedicated and standard keychain wrapper on ios.
Adding the note to the README seems like a good idea.